projects/packages/waf/run.php

```diff
@@ -1,24 +0,0 @@
-<?php
```

Comment:
Excellent point; we should update the code to generate a new standalone bootstrap on plugin update as well.

BTW @anomiex, do you know the answer here:
> At what point in the WordPress loading process do the module files like this run? Ideally, we want the WAF to run as soon as possible, is it directly after plugins_loaded?

Reply:
Module files are included during the `plugins_loaded` hook at a later priority - 100: https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/jetpack/class.jetpack.php#L918

Reply:
Not sure about the context of the question. What is "this"?

* Composer-run files like `action.php` get run when the autoloader is initialized. That's probably slightly before "plugins_loaded".
* The `run.php` was being run either from the "plugins_loaded" hook or from the `bootstrap.php` you generate for standalone mode. In the latter case I don't know when that happens.
* Looks like `projects/plugins/jetpack/modules/waf.php` will get loaded during the "plugins_loaded" hook, but [at the "100" priority](https://github.com/Automattic/jetpack/blob/5d2e51147c47500d9624cb87d25cf4e2767eb3e1/projects/plugins/jetpack/class.jetpack.php#L918) rather than the default "10".

Reply:
That does answer the question - since this is a firewall, we would like to run the WAF as early in the process as possible. Are you aware of a better way to do this? We are still new to WordPress and Jetpack plugin development.